Cargo.toml

```diff
@@ -5,3 +5,14 @@ authors = ["Casey Rodarmor <casey@rodarmor.com>", "Sönke Hahn <soenkehahn@gmail
 edition = "2018"
 
 [dependencies]
+maud = "0.22.2"
```

Comment:
How about an empty line after this line, just for cosmetic purposes?

Reply:
Good catch. This was `cargo add`'s doing.